GiEnJul/ClientApp/src/components/registerGiver/LocationGiver.tsx

```diff
@@ -1,99 +1,137 @@
-import * as React from 'react';
-import { Route } from 'react-router-dom'
-import useStyles from './Styles';
-import { ValidatorForm, SelectValidator } from 'react-material-ui-form-validator';
-import { Button, Grid, MenuItem, Container } from '@material-ui/core';
+import * as React from "react";
+import { Route } from "react-router-dom";
+import useStyles from "./Styles";
+import {
+  ValidatorForm,
+  SelectValidator,
+} from "react-material-ui-form-validator";
+import { Button, Grid, MenuItem, Container } from "@material-ui/core";
+import IGiverInputs from "./IGiverInputs";
+import { FAMILY_SIZES } from "../../common/constants/FamilySizes";
+import LOCATIONS from "../../common/constants/Locations";
 
-type Props = {
-  nextStep: () => void,
-  prevStep?: () => void,
-  handleLocationChange: (event: any) => void,
-  values: { location?: string; fullname?: string; email?: string; phoneNumber?: string; maxRecievers?: number; familyType?: string }
-  options: string[],
-  placeHolder: string,
+interface Props {
+  nextStep: () => void;
+  prevStep?: () => void;
+  values: IGiverInputs;
+  handleLocationChange: (event: any) => void;
+  placeHolder: string;
 }
-const LocationGiver: React.FC<Props> = ({ nextStep, prevStep, handleLocationChange, values, options, placeHolder }) => {
 
+const LocationGiver: React.FC<Props> = ({
```

Comment:
LocationGiver should be renamed, as this component renders both Familiy-selection and Location-selection

Reply:
I agree. I think these two forms should be two different components.